adapters/gamma/params_test.go

```diff
@@ -0,0 +1,60 @@
+package gamma
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+// This file actually intends to test static/bidder-params/brightroll.json
```

Comment:
Just a minor thing but can you please update this comment to point to `static/bidder-params/gamma.json`? Next line needs a similar update also